addons/etcd/dataprotection/restore.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+set -ex
+
+export PATH="$PATH:$DP_DATASAFED_BIN_PATH"
+export DATASAFED_BACKEND_BASE_PATH="$DP_BACKUP_BASE_PATH"
+
+DATA_DIR=/data
+mkdir -p ${DATA_DIR}
+res=$(find ${DATA_DIR} -type f)
+data_protection_file=${DATA_DIR}/.kb-data-protection
```

Comment:
I'll have a check to see whether data_protection_file is necessary.

Reply:
cleanup